server/main.lua

```diff
@@ -36,18 +36,30 @@ lib.callback.register('qbx_vehiclesales:server:spawnVehicle', function (source,
     return netId
 end)
 
+lib.callback.register('qbx_vehiclesales:server:checkVehicleOwner', function(source, plate)
+    local player = exports.qbx_core:GetPlayer(source)
+    local result = MySQL.single.await('SELECT * FROM player_vehicles WHERE plate = ? AND citizenid = ?', {plate, player.PlayerData.citizenid})

```

Comment:
Should call qbx_vehicles API instead of direct DB call here. Fine to leave for a future PR.

Reply:
Weird 🤔

Reply:
Oh well, I'll leave it for a future pr.